README.md

```diff
@@ -20,16 +20,39 @@
 
 ## Setup
 
-This project requires Ruby 2.5.3.
+This project requires Ruby 2.7.2.
 
 * Fork this repository
 * Clone your fork
 * From the command line, install gems and set up your DB:
     * `bundle`
-    * `rails db:create`
+    * `rake load_csv``
 * Run the test suite with `bundle exec rspec`.
 * Run your development server with `rails s` to see the app in action.
 
+#### A Note on Using Existing Data
+
+This project comes preloaded with customer, invoice, merchant, item and transaction data to pre-populate the application. The full data-set can be found in csv files under `app/db/data`. Additionally there is a collection of truncated data that is a lot easier to load and utilize that can be found at `app/db/data/test_data`. 
+After running `bundle install`, either `rake load_csv` or `rake load_test_csv` should be run. The rake files will create the database, apply migrations and then load csv data into your database.
```

Comment:
Do we want to specify to the user to use the `RAILS_ENV=development` and/or `RAILS_ENV=test`.